samples/native-lib/test_add.c

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2019 Intel Corporation.  All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "wasm_export.h"
+
+static int
+test_add_wrapper(wasm_exec_env_t *exec_env, int x, int y)
+{
+    return x + y;
+}
+
+/* clang-format off */
+#define REG_NATIVE_FUNC(func_name, signature) \
+    { #func_name, func_name##_wrapper, signature, NULL }
+
+static NativeSymbol native_symbols[] = {
+    REG_NATIVE_FUNC(test_add, "(ii)i")
```

Comment:
there should be a function named `test_add`, right?

Reply:
Yes, see file samples/native-lib/wasm-app/main.c: https://github.com/bytecodealliance/wasm-micro-runtime/pull/1120/files#diff-bc10ed7477a7549d93f8aca9e41f1fb832c68d4939f955bfe9f29c9769a521ff
it is compiled into wasm file, which calls to the imported function test_add/test_sqrt in its main function.